package.json

```diff
@@ -46,7 +46,6 @@
     "git-rev": "git://github.com/bewest/git-rev.git"
   },
   "devDependencies": {
-    "coveralls": "~2.11.2",
```

Comment:
Coveralls as I understand it is just a service to comment on github PRs and provide coverage badges on READMEs. What else are you looking for? You'll still get the coverage information when you run `make travis` or `instanbul cover...` locally.

See the `.travis.yml` file in the Express repo for an [example](https://github.com/strongloop/express/blob/master/.travis.yml) of this approach. (Note that Express installs a specific version of coveralls while I prefer to always install the latest. I'm open to dissent on this approach.)